META.list

```diff
@@ -853,3 +853,4 @@ https://raw.githubusercontent.com/tony-o/perl6-libcsv/master/META6.json
 https://raw.githubusercontent.com/alabamenhu/Fluent/master/META6.json
 https://raw.githubusercontent.com/bduggan/p6-Grammar-PrettyErrors/master/META6.json
 https://raw.githubusercontent.com/ALANVF/P6TK/master/META6.json
+https://gitlab.com/lembark/Perl6-FindBin/blob/master/META6.json
```

Comment:
You need to put the path to the raw JSON file.

Reply:
Sanity check: The URL references the META6.json file?

wget and firefox both snag the JSON for me.

Q: What is wrong with the URL?
Q: Other than an https-link to the META6.json file, what do I need to include?

Reply:
I think it needs to be this link: https://gitlab.com/lembark/Perl6-FindBin/raw/master/META6.json

Reply:
@lembark excuse my butting in here randomly, but I've sent you two mails from one account and another from a different account and I'd like to hear whether they are arriving or getting lost. thanks!

Reply:
Also, I see `CPAN:lembark`. If you upload your module to CPAN, then nothing needs to be changed in this repository. http://modules.perl6.org/ and zef will pick up your module automatically (although with some delay, up to 1 day I think).

Reply:
@AlexDaniel I have a question. Will http://modules.perl6.org and zef index this module if the META6.json is wrongly formatted or if any of the property in the file doesn't work? (like incorrect URL for the source-url property)?